.github/workflows/publish.yml

```diff
@@ -6,26 +6,42 @@ jobs:
     strategy:
       fail-fast: false
       matrix: # Windows-2022 has been temporarily removed. We are unable to build 2x windows installs at the same name due to naming convention issues. Issue made upstream here: https://github.com/tauri-apps/tauri-action/issues/215
-        platform: [macos-latest, ubuntu-latest, windows-2019]
+        os:
+          - ubuntu-latest
+          - macos-latest
+          - windows-2019
+          - windows-2022 # test again to see if it will fail, comment out this again if necessary
```

Comment:
We already discussed that this will make 2 Windows builds competing with each other and only one will win, but we don't know which one.

Reply:
yeah, since it was some time ago, I thought maybe something is changed 
I went to the issue link, but there was no update :(

Reply:
So, we should have 2 publish actions then? 
1 for windows2019, macos and linux
1 for windows2022
?

Reply:
It builds executable with the same name regardless of number of actions. An alternative would be to build executable without uploading it, renaming it and uploading renamed version (separate for Windows 10 and 11). This is all in case it is even needed in the first place (namely if desktop app doesn't run on both otherwise).